packages/mantine/src/theme/Theme.tsx

```diff
@@ -181,8 +182,8 @@ export const plasmaTheme: MantineThemeOverride = createTheme({
                 if (props.readOnly && !props.disabled) {
                     return {
                         root: {
-                            '--checkbox-icon-color': theme.colors.gray[7],
-                            '--checkbox-color': theme.colors.gray[2],
+                            '--checkbox-icon-color': 'var(--mantine-color-black)',
+                            '--checkbox-color': theme.colors.gray[1],
```

Comment:
This variable could be in the `plasmaCSSVariablesResolver.ts` in the `// custom color` section. We could name it `'--coveo-color-input-bg-readonly'` so we could reuse it in every other inputs ?

Reply:
added the variables to the variable resolver and custom style to achieve the desired look, I think we should do the same for the other components until we have a cleaner way to do this